src/Pie.js

```diff
@@ -3,9 +3,28 @@ import PropTypes from 'prop-types';
 import { Platform } from 'react-native';
 import { Surface, Shape, Path, Group } from '@react-native-community/art';
 
-function createPath(cx, cy, r, startAngle, arcAngle) {
+function createPath(cx, cy, r, startAngle, arcAngle, isBezian, innerRadius) {
   const p = new Path();
-  if (Platform.OS === 'web') {
+  //starting point of our chart
+  if(isBezian && Platform.OS !== 'web'){
+    let ROUNDNESSOUSIDE = 1 - ((r - innerRadius)/ innerRadius) - (arcAngle * .5);
+    let ROUNDNESSINSIDE = 1 + ((r - innerRadius)/ innerRadius) + (arcAngle * .5);
```

Comment:
Just curious.. why all caps? Because they are constants? 

We can make these `const`s, and do this case: `ROUNDNESS_OUTSIDE`..

What do you think @kevkevinpal ?

Reply:
Then again this isn't a convention followed on this repo so id, maybe camel case is better here. I'll leave it up to you